ui/components/app/collectibles-tab/collectibles-tab.js

```diff
@@ -58,48 +58,50 @@ export default function CollectiblesTab({ onAddNFT }) {
   }
 
   return (
-    <div className="collectibles-tab">
+    <Box className="collectibles-tab">
       {Object.keys(collections).length > 0 ||
       previouslyOwnedCollection.collectibles.length > 0 ? (
         <CollectiblesItems
           collections={collections}
           previouslyOwnedCollection={previouslyOwnedCollection}
         />
       ) : (
-        <Box padding={[6, 12, 6, 12]}>
+        <>
           {isMainnet &&
           !useCollectibleDetection &&
           !collectibleDetectionNoticeDismissed ? (
             <CollectiblesDetectionNotice />
           ) : null}
-          <Box justifyContent={JUSTIFY_CONTENT.CENTER}>
-            <img src="./images/no-nfts.svg" />
-          </Box>
-          <Box
-            marginTop={4}
-            marginBottom={12}
-            justifyContent={JUSTIFY_CONTENT.CENTER}
-            flexDirection={FLEX_DIRECTION.COLUMN}
-            className="collectibles-tab__link"
-          >
-            <Typography
-              color={COLORS.UI3}
-              variant={TYPOGRAPHY.H4}
-              align={TEXT_ALIGN.CENTER}
-              fontWeight={FONT_WEIGHT.BOLD}
+          <Box padding={12}>
+            <Box justifyContent={JUSTIFY_CONTENT.CENTER}>
+              <img src="./images/no-nfts.svg" />
+            </Box>
+            <Box
+              marginTop={4}
+              marginBottom={12}
+              justifyContent={JUSTIFY_CONTENT.CENTER}
+              flexDirection={FLEX_DIRECTION.COLUMN}
+              className="collectibles-tab__link"
             >
-              {t('noNFTs')}
-            </Typography>
-            <Button
-              type="link"
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://metamask.zendesk.com/hc/en-us/articles/360058238591-NFT-tokens-in-MetaMask-wallet"
-            >
-              {t('learnMoreUpperCase')}
-            </Button>
+              <Typography
+                color={COLORS.UI3}
```

Comment:
```suggestion
                color={COLORS.TEXT_MUTED}
```

Reply:
Why was this suggestion ignored?
I think this isn't the desired color for this text (what I'm seeing now):
![Screen Shot 2022-03-28 at 9 00 20 AM](https://user-images.githubusercontent.com/34557516/160414658-275515d1-a83b-4dd1-9267-b61a216c208f.png)
?

Reply:
Pretty sure we want "No NFTs yet" to look like:
![Screen Shot 2022-03-28 at 9 20 53 AM](https://user-images.githubusercontent.com/34557516/160419002-2479b58d-9213-45b1-b4d8-af93393945b1.png)

Reply:
ping here @NiranjanaBinoy

Reply:
I am not sure how I missed it, it might have been lost during rebasing and conflict resolution. I have updated that now.